SingularityService/src/main/java/com/hubspot/singularity/data/SingularityValidator.java

```diff
@@ -436,4 +437,12 @@ private boolean isValidInteger(String strValue) {
       return false;
     }
   }
+
+  public static String encodeZkName(String name) {
```

Comment:
All our other method names are named in the format `check...`, maybe `checkUserId` or `checkAndEncodeUserId` to show what's being returned + be consistent with other methods in the class